rebasehelper/utils.py

```diff
@@ -39,6 +39,10 @@
 import pyquery
 import hashlib
 import requests
+import termios
+import fcntl
+import tty
+import datetime
```

Comment:
The imports are all wrong. Since you are adding a lot of new ones, you could as well fix that. Third party imports are mixed with standard library ones and the order is not alphabetical, which is a pain considering the number of them.